infrastructure/ansible/roles/traffic_portal/defaults/main.yml

```diff
@@ -208,7 +208,7 @@ tp_default_properties_template:
         _comment: Do you want to show a charts button for each server? If so, where
           can the charts be found? xml id will be appended to the provided url.
         show: false
-        baseUrl: "{{ tp_ts_base_url }}/dashboard/script/traffic_ops_server.js?which="
+        baseUrl: "{{ tp_ts_base_url }}/dashboard/db/server-stats?var-SERVER="
```

Comment:
`traffic_ops_server.js` is in our repo at [/traffic_stats/grafana/traffic_ops_server.js](https://github.com/apache/trafficcontrol/blob/048a1ea016/traffic_stats/grafana/traffic_ops_server.js). If the `cdn-overview` dashboard is not publicly available, I would be hesitant to reference it here.

Reply:
Unless we are ready to remove the existing grafana scripts from the repo, I don't think we even need to change these URLs in this PR, since they already can be overridden in an org-specific repo that has its own dashboards.

Reply:
i think zach is right